presto-docs/src/main/sphinx/release/release-0.243.rst

```diff
@@ -0,0 +1,48 @@
+=============
+Release 0.243
+=============
+
+**Highlights**
+==============
+* Add function ``truncate(x, n)``.
```

Comment:
You should link to the documentation for these functions.  And maybe give a brief summary here of what they do. (and same for the actual notes below)